lib/shipit.rb

```diff
@@ -140,6 +140,14 @@ def host
     secrets.host.presence
   end
 
+  def enforce_publish_config
+    secrets.enforce_publish_config.presence ? secrets.enforce_publish_config : nil
+  end
+
+  def private_npm_registry
+    secrets.private_npm_registry.presence ? secrets.private_npm_registry : nil
```

Comment:
This is redundant, `#presence` returns `private_npm_registry` or `nil` already.

Reply:
Neat, TIL!